opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -1144,6 +1148,25 @@ def error_status_test(context):
             .start_as_current_span("root")
         )
 
+    def test_record_exception_fqn(self):
+        span = trace._Span("name", mock.Mock(spec=trace_api.SpanContext))
+        exception = DummyError("error")
+        exception_type = f"{exception.__module__}.{type(exception).__qualname__}"
```

Comment:
This would be better to hardcode that to reimplement what the tested code is doing

Reply:
Agreed. Let me put the full path